content/android/_index.md

```diff
@@ -20,3 +20,5 @@ draft: false
 - [GitHub Issues](https://github.com/NASAWorldWind/WorldWindAndroid/issues/) lists WorldWind bugs, issues, and
         milestones
 - [Forum](https://forum.worldwindcentral.com) provides help from the WorldWind community
+
+---
```

Comment:
Did you mean to leave the trailing horizontal line?

Reply:
My apologies, I'll remove it!